targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/TARGET_SDT64B/PinNames.h

```diff
@@ -0,0 +1,311 @@
+    PTD16 = (3 << GPIO_PORT_SHIFT | 16),
+    PTD17 = (3 << GPIO_PORT_SHIFT | 17),
+    PTD18 = (3 << GPIO_PORT_SHIFT | 18),
+    PTD19 = (3 << GPIO_PORT_SHIFT | 19),
+    PTD20 = (3 << GPIO_PORT_SHIFT | 20),
+    PTD21 = (3 << GPIO_PORT_SHIFT | 21),
+    PTD22 = (3 << GPIO_PORT_SHIFT | 22),
+    PTD23 = (3 << GPIO_PORT_SHIFT | 23),
+    PTD24 = (3 << GPIO_PORT_SHIFT | 24),
+    PTD25 = (3 << GPIO_PORT_SHIFT | 25),
+    PTD26 = (3 << GPIO_PORT_SHIFT | 26),
+    PTD27 = (3 << GPIO_PORT_SHIFT | 27),
+    PTD28 = (3 << GPIO_PORT_SHIFT | 28),
+    PTD29 = (3 << GPIO_PORT_SHIFT | 29),
+    PTD30 = (3 << GPIO_PORT_SHIFT | 30),
+    PTD31 = (3 << GPIO_PORT_SHIFT | 31),
+    PTE0  = (4 << GPIO_PORT_SHIFT | 0 ),
+    PTE1  = (4 << GPIO_PORT_SHIFT | 1 ),
+    PTE2  = (4 << GPIO_PORT_SHIFT | 2 ),
+    PTE3  = (4 << GPIO_PORT_SHIFT | 3 ),
+    PTE4  = (4 << GPIO_PORT_SHIFT | 4 ),
+    PTE5  = (4 << GPIO_PORT_SHIFT | 5 ),
+    PTE6  = (4 << GPIO_PORT_SHIFT | 6 ),
+    PTE7  = (4 << GPIO_PORT_SHIFT | 7 ),
+    PTE8  = (4 << GPIO_PORT_SHIFT | 8 ),
+    PTE9  = (4 << GPIO_PORT_SHIFT | 9 ),
+    PTE10 = (4 << GPIO_PORT_SHIFT | 10),
+    PTE11 = (4 << GPIO_PORT_SHIFT | 11),
+    PTE12 = (4 << GPIO_PORT_SHIFT | 12),
+    PTE13 = (4 << GPIO_PORT_SHIFT | 13),
+    PTE14 = (4 << GPIO_PORT_SHIFT | 14),
+    PTE15 = (4 << GPIO_PORT_SHIFT | 15),
+    PTE16 = (4 << GPIO_PORT_SHIFT | 16),
+    PTE17 = (4 << GPIO_PORT_SHIFT | 17),
+    PTE18 = (4 << GPIO_PORT_SHIFT | 18),
+    PTE19 = (4 << GPIO_PORT_SHIFT | 19),
+    PTE20 = (4 << GPIO_PORT_SHIFT | 20),
+    PTE21 = (4 << GPIO_PORT_SHIFT | 21),
+    PTE22 = (4 << GPIO_PORT_SHIFT | 22),
+    PTE23 = (4 << GPIO_PORT_SHIFT | 23),
+    PTE24 = (4 << GPIO_PORT_SHIFT | 24),
+    PTE25 = (4 << GPIO_PORT_SHIFT | 25),
+    PTE26 = (4 << GPIO_PORT_SHIFT | 26),
+    PTE27 = (4 << GPIO_PORT_SHIFT | 27),
+    PTE28 = (4 << GPIO_PORT_SHIFT | 28),
+    PTE29 = (4 << GPIO_PORT_SHIFT | 29),
+    PTE30 = (4 << GPIO_PORT_SHIFT | 30),
+    PTE31 = (4 << GPIO_PORT_SHIFT | 31),
+
+       // Analog
+    A0 = PTB6,
+    A1 = PTB7,
+    A2 = DAC0_OUT,
+    //A3 = DAC1_OUT,
+
+    // General Pin Input Output (GPIO)
+    GPIO0 = PTC1,
+    GPIO1 = PTC5,
+    GPIO2 = PTD6,
+    GPIO3 = PTC9,
+    GPIO4 = PTC3,
+    GPIO5 = PTC6,
+    GPIO6 = NOT_CONNECTED,
+
+    //Purse Width Modulation (PWM)
+    PWM0 = GPIO2,
+    PWM1 = GPIO3,
+    PWM2 = GPIO0,
+    PWM3 = GPIO1,
+
+    // LEDs
+    LED0 = GPIO0,
+    LED1 = GPIO1,
+    LED2 = GPIO2,
+   
+    LED_RED   = LED0,
+    LED_GREEN = LED1,
+    LED_BLUE  = LED2,
+
+    // USB bridge and SWD UART connected UART pins
+    USBTX = PTC15,
+    USBRX = PTC14,
+
+    // UART pins
+    UART0_RX  = PTD8,
+    UART0_TX  = PTD9,
+    UART0_CTS = PTD11,
+    UART0_RTS = PTD10,
+
+    UART1_RX  = USBRX,
+    UART1_TX  = USBTX,
+    UART1_CTS = PTC13,
+    UART1_RTS = PTC12,
+
+    UART2_RX  = PTC16,
+    UART2_TX  = PTC17,
+    UART2_CTS = PTC19,
+    UART2_RTS = PTC18,
+
```

Comment:
Can you please define default peripherals for each type? Ex: UART_TX = UART0_TX , I2C_SDA = I2C1_SDA and so on..

Reply:
Where can I find the names for default peripherals? I see LED1, 2, 3, and USBTX, RX, but others don't seem to have that. For example, https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_Maxim/TARGET_MAX32620C/TARGET_MAX32620FTHR/PinNames.h

Reply:
@ashok-rao I'll define the following list:
USBTX, USBTX, LED1, LED2, LED3, LED_RED, LED_GREEN, LED_BLUE, SERIAL_TX, SERIAL_RX, I2C_SCL, I2C_SDA, SPI_MOSI, SPI_MISO, SPI_SCK, SPI_CS, PWM_OUT

Anything else?

Reply:
That should do it! Thanks!